### PR TITLE
dartValue/jsValue => convertToDart/convertToJs

### PR DESCRIPTION
**Pull request by jakemac53**

### CODE REVIEW

File: test/convert_test.dart [hunk 1]
```diff
@@ -32,15 +32,15 @@ main() {
...
+        var array = convertToJs(list) as JsArray;
+        expect(convertToDart(array), list);
       });
 
       test('object', () {
         var model = new MyModel();
         var object =
-            new JsObject.jsify({'1': 1, 'model': jsValue(model), 'a': 'a',});
-        var dartMap = dartValue(object) as Map;
+            new JsObject.jsify({'1': 1, 'model': convertToJs(model), 'a': 'a',});
```
Comment (sigmundch) [inline test/convert_test.dart]: I can't tell in github PRs whether the lines got too long, so if you haven't you may want to run dartfmt

**Comment by sigmundch** [comment]
lgtm

File: test/convert_test.dart (hunk 1 above)
Comment (jakemac53) [inline test/convert_test.dart]: done

**Comment by jakemac53** [comment]
Actually just pushed this as `0.2.0-rc1` since its a breaking change

**Pull request closed without merging**